app/controllers/dashboard_controller.rb

```diff
@@ -58,19 +58,42 @@ def user_profile
                     Please try again."
   end
 
+<<<<<<< HEAD
+<<<<<<< HEAD
+=======
+>>>>>>> a978778... Implement a redirect for user who are not signed in who want to view other users' profile.
   def profile_view
     @user = User.find(profile_params[:taskee_id])
   end
 
+<<<<<<< HEAD
   def assign_task
   end
 
+=======
+>>>>>>> 7bf8900... Implement the profile settings page on dashboard
+=======
+  # def anonymous_viewer
+
+  # end
+
+>>>>>>> a978778... Implement a redirect for user who are not signed in who want to view other users' profile.
   private
 
   def quiz_params
     params.permit(:aced)
   end
 
+<<<<<<< HEAD
+<<<<<<< HEAD
+=======
+  def profile_view
+    @user = User.find(profile_params[:taskee_id])
+  end
+
+>>>>>>> 7bf8900... Implement the profile settings page on dashboard
+=======
+>>>>>>> a978778... Implement a redirect for user who are not signed in who want to view other users' profile.
```

Comment:
unexpected token tIDENTIFIER
(Using Ruby 2.3 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)<br>unterminated string meets end of file
(Using Ruby 2.3 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)